instagram-app/src/components/PostContainer/messagesection/messagesection.js

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import './messagesection.css';
+
+const Messagesection = props => {
+    return (
+        <div className="messagesection_class">
+            <div>
+                <p className="user_name_class"><a href="https://cocacola.com">{props.message.username}</a><span style={{fontWeight:'200', marginleft:'7px'}}>{props.message.text}</span></p>
```

Comment:
Not sure what the web address is here. My browser doesn't like it and tells me to go back to safety. LOL. I don't think you need it here?